internal/http/fireblocks_proposal.go

```diff
@@ -34,14 +34,16 @@ type FireblocksProposal struct {
 func NewFireblocksProposal() *FireblocksProposal {
 	// use HDWallet 44/1/1/0/0, VaultAccountId:1
 	assetId := "BTC_TEST"
+	vaultAccountId := "1"
 	if types.GetBTCNetwork(config.AppConfig.BTCNetworkType).Name == "mainnet" {
 		assetId = "BTC"
+		vaultAccountId = "6"
 	}
 	return &FireblocksProposal{
 		Bip44AddressIndex: 1,
```

Comment:
set to vaultAccountId, also DerivationPath: [5]uint32{44, 1, vaultAccountId, 0, 0},

Reply:
fixed